internal/resolution/variablesources/bundle_test.go

```diff
@@ -4,19 +4,196 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/operator-framework/deppy/pkg/deppy"
 	"github.com/operator-framework/operator-registry/alpha/declcfg"
 	"github.com/operator-framework/operator-registry/alpha/property"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	"github.com/operator-framework/operator-controller/internal/catalogmetadata"
 	olmvariables "github.com/operator-framework/operator-controller/internal/resolution/variables"
 	"github.com/operator-framework/operator-controller/internal/resolution/variablesources"
 	testutil "github.com/operator-framework/operator-controller/test/util"
 )
 
+func TestMakeBundleVariables(t *testing.T) {
+	t.Run("valid dependencies", func(t *testing.T) {
```

Comment:
Since the `t.Run()`s here have nothing shared, the typical pattern I've seen is to split them into separate top-level `Test*` functions:

```go
func TestMakeBundleVariables_ValidDepedencies(t *testing.T) {}
func TestMakeBundleVariables_NonExistentDepedencies(t *testing.T) {}
```